app/templates/dashboard_all.html

```diff
@@ -0,0 +1,48 @@
+{%- from "govuk_frontend_jinja/components/tag/macro.html" import govukTag -%}
+{%- from "govuk_frontend_jinja/components/table/macro.html" import govukTable -%}
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+{% from "partials/start-application-button.html" import startApplicationButton %}
+{% from "partials/available_in_language.html" import available_in_language %}
+{% from "partials/round_closed_warning.html" import round_closed_warning %}
+{% from "partials/applications_table.html" import applications_table %}
+
+{% extends "base.html" %}
+{% set pageHeading %}
+{% trans %}All applications{% endtrans %}
+{% endset %}
+{% block content %}
+    <h1 class="govuk-heading-xl">{{ pageHeading }}</h1>
+    <p class="govuk-body">
+        {% set application_count=display_data["total_applications_to_display"]%}
+        {% trans %}You have started{% endtrans %}&nbsp;{% trans count=application_count%} {{application_count}} application{% pluralize %}{{application_count}} applications{% endtrans %}&nbsp;{% trans %}using this email address{% endtrans %}.
+    </p>
+{{available_in_language(inset=False, warn=True)}}
+
+<div class="govuk-accordion" data-module="govuk-accordion" id="accordion-default">
+    {% for fund in display_data["funds"] %}
+        <div class="govuk-accordion__section">
+            <div class="govuk-accordion__section-header">
+                <h2 class="govuk-accordion__section-heading">
+                    <span class="govuk-accordion__section-button" id="funds-heading-{{fund['fund_data']['short_name']}}">
+                        {{ fund["fund_data"]["name"] }}
+                    </span>
+                </h2>
+            </div>
+            <div id="funds-content-{{fund['fund_data']['short_name']}}" class="govuk-accordion__section-content" aria-labelledby="funds-heading-{{fund['fund_data']['short_name']}}">
+
+                {% for round in fund["rounds"] %}
+                    {% if not round["is_not_yet_open"] and not round["is_past_submission_deadline"] %}
+                        <span class="govuk-caption-m">{{ round["round_details"]["title"] }}</span>
+                        {% if round["is_past_submission_deadline"] %}
```

Comment:
I think this code in unreachable due to the first if statement. 
the only way to enter the if is if is_past_submission_deadline == False, so`and not False` on line 34 will get into the first block 
then `if False` on line 36 will prevent the second block ever executing

Reply:
you're right! have removed this now 👍🏾